packages/dara-components/js/smart/chat/chat.tsx

```diff
@@ -30,26 +30,32 @@ interface MessageNotificationPayload {
     content: Message;
 }
 
-const ChatWrapper = styled.div`
+const ThreadWrapper = styled.div`
     pointer-events: auto;
 
     position: fixed;
-    z-index: 999;
     right: 1rem;
     bottom: -0.1rem;
+
+    margin: 1rem;
+
+    border-radius: 0.4rem;
 `;
 
 const ChatButton = styled.button`
-    width: 3.5rem;
-    height: 3.5rem;
-    margin: 0 2rem 2rem 0;
-    padding-top: 0.5rem;
+    position: absolute;
```

Comment:
Do all our layout components have `position: relative` for this to attach to?

Reply:
Yes, in the definition of the class for LayoutComponent we set the position to relative for all of them